src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -233,20 +233,31 @@ public function assets() {
 
 		tribe_asset(
 			$plugin,
-			'tribe-tickets-rsvp',
-			'v2/rsvp.js',
-			[ 'jquery' ],
+			'tribe-tickets-rsvp-manager',
+			'v2/rsvp-manager.js',
+			[ 'jquery', 'tribe-common', 'tribe-tickets-rsvp-block' ],
 			null,
 			[
 				'localize' => [
 					'name' => 'TribeRsvp',
 					'data' => [
 						'ajaxurl' => admin_url( 'admin-ajax.php', ( is_ssl() ? 'https' : 'http' ) ),
+						'cancelText' => esc_js( 'Are you sure you want to cancel?', 'event-tickets' ),
```

Comment:
:no_entry_sign: Barf out, gag me with a spoon! esc_js()!This is for inline javascript, which is against our standards. Use json_encode instead.

<sub>hash: 680b0768aec8f0634762f602899dfc62</sub>

Reply:
Valid bot complaint.

Reply:
this should be translated

Reply:
🤦‍♂️Thanks! yeah, pushed the code and forgot this one.